tests/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
-set -x
+set -u
+set -o pipefail
+#set -x
```

Comment:
I personally liked having `set -x` despite its verbosity as it allowed me to see easily where an error may originate in a CI system.

Reply:
Ok. I've reverted these lines.